pkg/api/v1/types.go

```diff
@@ -3404,6 +3404,15 @@ type PodExecOptions struct {
 	Command []string `json:"command" protobuf:"bytes,6,rep,name=command"`
 }
 
+// PodPortForwardOptions is the query options to a Pod's port forward call.
+type PodPortForwardOptions struct {
+	metav1.TypeMeta `json:",inline"`
+
+	// List of ports to forward
```

Comment:
If this is required indicate so in the Godoc.  Also use the form where you include the external name (`ports`) in the Godoc.